File: FEB/Palindrome.c
#include<stdio.h>

void Display(int No)
{
    int rem=0,sum=0;
    int temp=0;
    temp=No;
    while(No!=0)
    {
        rem=No%10;
        sum=sum*10+rem;
        No=No/10;

    }
    No=temp;
    if(sum==No)
    {
        printf("Number is Palindrome :");
    }
    else
    {
        printf("Number is Not Palindrome :");
    }

}
int main()
{
    int ino=0;
    printf("Enter the number: ");
    scanf("%d",&ino);
    Display(ino);
    return 0;
}

File: New/Number/Palindrome.java
import java.util.*;;
public class Palindrome {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = sc.nextInt();
        int sum=0;
        int temp=n;
        while(n!=0)
        {
            int rem=n%10;
            sum=sum*10+rem;
            n=n/10;

        }
        if(temp==sum)
        {
            System.out.print("Number is Palindrome: ");
        }
        else
        {
            System.out.print("Number is Not  Palindrome: ");
        }

    }
    
}
